### PR TITLE
use dotenv to manage environment variables

### PR DESCRIPTION
**Pull request by cspalding**
Added a `.env.example` to source control with the variables that are necessary for setup, updated setup instructions to include how to update the `.env` file to work locally.

The next PR will introduce either a Rack middleware or some other thin wrapper for the feature hub client to make getting flags easier!

### CODE REVIEW

File: examples/rails_example/.env.example [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# running via docker? use this URL:
+# "http://host.docker.internal:8085/",
+FEATUREHUB_EDGE_URL="http://0.0.0.0:8085/"
+# You can find this value in the admin console by navigating to the 'API Keys' section of the UI and choosing
+# between the client and the server API Keys for whatever service account you've set up. Learn more about the
+# differences between client-side evaluation here:
+# https://docs.featurehub.io/featurehub/latest/sdks-development.html#_client_side_evaluation
+# and server-side evaluation here (though it's almost never recommended):
+# https://docs.featurehub.io/featurehub/latest/sdks-development.html#_supporting_server_side_evaluation
+FEATUREHUB_CLIENT_API_KEY="CHANGE_ME"
```
Comment (cspalding) [inline examples/rails_example/.env.example:13]: Just copy/pasted this stuff from the initializer

File: examples/rails_example/README.md [hunk 2]
```diff
@@ -19,7 +19,8 @@ running `bundle install`.
 
 ## FeatureHub Setup
 
-In order for this app to work locally you'll need to have FeatureHub set up on your machine. In order to get the simplest version of FeatureHub working, run the command from [Evaluating FeatureHub](https://docs.featurehub.io/featurehub/latest/index.html#_evaluating_featurehub).
+In order for this app to work locally you'll need to have FeatureHub set up on your machine or have an instance running somewhere.
```
Comment (cspalding) [inline examples/rails_example/README.md:22]: You mentioned this in a comment on the last PR :)

**Comment by cspalding** [comment]
@rvowles sorry to bother you; if you have a few minutes to review I would appreciate it!

**Review by rvowles — APPROVED** [review]

**Comment by rvowles** [comment]
Sorry Chris, flurry of activity around that time and this got lost in my email. Thanks for pinging me again! Ok to merge?

**Comment by cspalding** [comment]
> Sorry Chris, flurry of activity around that time and this got lost in my email. Thanks for pinging me again! Ok to merge?

No worries at all! This one is good to merge 👍🏻

**Pull request merged**